R/calculate_silhouette_width.R

```diff
@@ -30,8 +31,8 @@ calculate_silhouette_width <- function(merged_sce,
   set.seed(seed)
```

Comment:
I am not sure that we still need to set a seed here, given that it is set fresh every time `silhouette_with_from_pcs` is called. I do think it is better to set the seed from that latter function, to ensure that seed is being used for all random sampling and not something later in the random number generator chain.
Anyways, since we set the seed later, this doesn't _hurt_ anything to stick around, but it also doesn't _do_ anything. So, it can go.